code/edgeconsent/src/main/java/com/adobe/marketing/mobile/edge/consent/Consents.java

```diff
@@ -134,11 +135,51 @@ void merge(final Consents newConsents) {
 		}
 
 		if (isEmpty()) {
-			consentsMap = newConsents.consentsMap;
+			consentsMap = Utils.optDeepCopy(newConsents.consentsMap, new HashMap<>());
```

Comment:
nit: extract `Utils.optDeepCopy(newConsents.consentsMap, new HashMap<>());` to make the code less verbose.